tracpro/settings/dev.py

```diff
@@ -3,7 +3,7 @@
 from .base import *  # noqa
 
 
-CELERY_ALWAYS_EAGER = True
+CELERY_ALWAYS_EAGER = False
```

Comment:
Let's not commit this change. Although it masked this particular error, using `CELERY_ALWAYS_EAGER = True` is usually a benefit for development.